packages/0xcert-scaffold/src/core/asset-ledger.ts

```diff
@@ -343,9 +357,16 @@ export interface AssetLedgerObjectUpdateRecipe {
 export interface AssetLedgerUpdateRecipe {
 
   /**
-   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended foo each asset.
+   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended for each asset.
    * Example: https://example.com/id/
-   * Asset 1 URI will become: https://example.com/id/1
+   * Asset 1 URI will become: https://example.com/id/1 + postfix
+   */
+  uriPrefix: string;
+
+  /**
+   * Uri postfix for metadata URI-s. After uriPrefix and assetId postfix is automatically appended for each asset.
```

Comment:
* URI postfix for metadata URIs. After uriPrefix and assetId, postfix is automatically appended for each asset.